lib/widgets/speaker_detail/speaker_detail_divider.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:flutter/material.dart';
+import 'package:hackathon_flutterturkiye_org/core/styles/theme_helper.dart';
+
+class SpeakerDetailDivider extends StatelessWidget {
```

Comment:
if this divider is a generic divider, let's rename it otherwise, let's put this inside the speaker detail widget as a private widget.

Reply:
I used this divider three times on different files. I will rename it.

Reply:
Why don't we use the Divider widget?

Reply:
> Why don't we use the Divider widget?

Because it didn't work. It was invisible.

For that reason, I created my own.